### ISSUE TITLE
A Question about Virtual Environment and sys.path

### ISSUE THREAD

**Issue by dr-GitHub-account** [open-mmlab/mmdetection#7333]
According to 
[https://mmdetection.readthedocs.io/en/v2.21.0/get_started.html#installation](url)
users are supposed to create a virtual environment openmmlab and then do the installation. I have already prepared the virtual environment and it worked. I noticed that when I printed elements in sys.path in a .py file and run this file under virtual environment openmmlab, I got the directory of this .py file, got paths under anaconda3/lib/python3.7. I also got path openmmlab/lib/python3.7/site-packages and some paths under it. The aforementioned paths seem reasonable since they are related to the .py file or the virtual environment. 

However, I also noticed .../mmdetection was printed and I wonder why. I wonder in which step of installation the virtual environment openmmlab become related to the directory .../mmdetection. Thank you!

**Comment by ZwwWayne** [comment]
Did you run this command? `PYTHONPATH="$(dirname $0)/..":$PYTHONPATH`, we add it in the script as a default.

**Comment by DrX2020** [comment]
I did not run this command explicitly.

Today I tried to create a new environment and print sys.path in a python script. Before I ran `python setup.py develop` in directory ./mmdetection, ./mmdetection was not in sys.path. After I ran `python setup.py develop`, ./mmdetection was added in sys.path, together with some .egg files under ./site-packages.

I am wondering does `python setup.py develop` has something to do with sys.path and the command you mentioned?
